### PR TITLE
TextBox: Safeguard against undefined inputAttr option

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: packages/devextreme/js/ui/text_box/ui.text_editor.base.js [hunk 1]
```diff
@@ -493,8 +493,8 @@ const TextEditorBase = Editor.inherit({
     },
 
     _setFieldAria(force) {
-        const { 'aria-label': ariaLabel } = this.option('inputAttr');
-
+        const inputAttributeOption = this.option('inputAttr');
```
Comment (marker-dao) [inline packages/devextreme/js/ui/text_box/ui.text_editor.base.js]:
```suggestion
        const inputAttr = this.option('inputAttr');
```

File: packages/devextreme/js/ui/text_box/ui.text_editor.base.js [hunk 2]
```diff
@@ -493,8 +493,8 @@ const TextEditorBase = Editor.inherit({
...
+        const ariaLabel = inputAttributeOption && inputAttributeOption['aria-label'];
```
Comment (marker-dao) [inline packages/devextreme/js/ui/text_box/ui.text_editor.base.js]: Try it pls

```suggestion
        const ariaLabel = inputAttr?.['aria-label'];
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js [hunk 3]
```diff
@@ -31,6 +31,17 @@ const BUTTONS_CONTAINER_CLASS = 'dx-texteditor-buttons-container';
 const TEXTEDITOR_INPUT_CONTAINER_CLASS = 'dx-texteditor-input-container';
 
 QUnit.module('common', {}, () => {
+    QUnit.test('should not generate any aria-label error if inputAttr = undefined', function(assert) {
```
Comment (marker-dao) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js]:
```suggestion
    QUnit.test('should not generate any error if inputAttr is undefined', function(assert) {
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js [hunk 4]
```diff
@@ -31,6 +31,17 @@ const BUTTONS_CONTAINER_CLASS = 'dx-texteditor-buttons-container';
...
+        try {
+            $('#textbox').dxTextBox({
+                inputAttr: undefined
+            });
+            assert.ok(true, 'Encountered no errors');
+        } catch(e) {
+            assert.ok(false, `the error is thrown: ${e.message}`);
```
Comment (marker-dao) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js]:
```suggestion
            assert.ok(false, `The error is thrown: ${e.message}`);
```

Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js]: [0c3f424](https://github.com/DevExpress/DevExtreme/pull/27328/commits/0c3f424ee03517e7723add3750e3182b0ebe7eff)

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js (hunk 3 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js]: [0c3f424](https://github.com/DevExpress/DevExtreme/pull/27328/commits/0c3f424ee03517e7723add3750e3182b0ebe7eff)

File: packages/devextreme/js/ui/text_box/ui.text_editor.base.js (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/ui/text_box/ui.text_editor.base.js]: this works tested it as well
[0c3f424](https://github.com/DevExpress/DevExtreme/pull/27328/commits/0c3f424ee03517e7723add3750e3182b0ebe7eff)

File: packages/devextreme/js/ui/text_box/ui.text_editor.base.js (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/js/ui/text_box/ui.text_editor.base.js]: [0c3f424](https://github.com/DevExpress/DevExtreme/pull/27328/commits/0c3f424ee03517e7723add3750e3182b0ebe7eff)

**Review by marker-dao — APPROVED** [review]

File: packages/devextreme/js/ui/text_box/ui.text_editor.base.js (hunk 1 above)
Comment (Zedwag) [reply packages/devextreme/js/ui/text_box/ui.text_editor.base.js]: in this case it whould be better to use `const { inputAttr } = this.option();` but let leave it as it is now

**Review by Zedwag — APPROVED** [review]

**Pull request merged**